src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.lang.Math;
+
 public class Duke {
```

Comment:
I like how you have used comments in the code to explain the logic and make it easier to understand. However, any reason why you structured the method this way instead of breaking it down into simpler methods or keeping the methods shorter and more readable? :worried: